qemu/tests/cfg/virtio_serial_file_transfer.cfg

```diff
@@ -43,6 +43,15 @@
             type = virtio_serial_file_transfer_max_ports
             start_vm = no
             virtio_serial_ports = 30
+        - offline_migration:
+            only unix_socket
+            type = virtio_serial_file_transfer_offline_migrate
+            serials += " vs2"
+            serial_type_vs2 = virtserialport
+            file_transfer_serial_port = vs1
```

Comment:
If test two ports,  remove this definition, traversal serials start with 'virt' in script is ok.

Reply:
> If test two ports, remove this definition, traversal serials start with 'virt' in script is ok.

update and thanks